hanabi.py

```diff
@@ -27,10 +34,19 @@ class Play:
 class Card:
     color: str
     number: int
+
+    def __str__(self):
+        return f"{self.color}{self.number}"
+
+@dataclass
+class CardInformation:
```

Comment:
I think it would be easier to simply merge the `CardInformation` and `Card` classes. This would be less bug-prone because rearranging cards becomes much simpler

Reply:
Also, you'd have to rename `color` -> `color_hints`, which I think would be a good for clarity anyways

Reply:
Should probably add a comment here that duck variant bots cannot access color or number attributes.

Reply:
I've added the 'Variant' classes to deal with this now. When playing `VariantDuck` the number/color information does not update, only the `clued` flag.